app/assets/javascripts/admin/assets_admin.js

```diff
@@ -10,6 +10,5 @@
 // WARNING: THE FIRST BLANK LINE MARKS THE END OF WHAT'S TO BE PROCESSED, ANY BLANK LINE SHOULD
 // GO AFTER THE REQUIRES BELOW.
 //
-//= require 'admin/jquery.iframe-transport'
-//= require 'admin/jquery.fileupload'
+//= require 'blueimp-file-upload/js/jquery.fileupload.js'
```

Comment:
replace fileupload with blueimp's npm dependency. iframe-transport is included in this, but we don't really need it because it's for xhr support in older browsers that we no longer support internally.